package.json

```diff
@@ -37,6 +37,7 @@
     "@babel/preset-typescript": "^7.3.3",
     "@elastic/eslint-config-kibana": "link:../../packages/eslint-config-kibana",
     "@elastic/eslint-import-resolver-kibana": "link:../../packages/kbn-eslint-import-resolver-kibana",
+    "@elastic/eui": "^22.3.1",
```

Comment:
Do we find out why need to use an old version of EUI? Kibana 7.9 is using "26.3.3"

Reply:
@ylwu-amzn I'm still researching, adding this allows most of UT to pass.

Reply:
@ylwu-amzn the upgrade from 22.3.1 to 26.3.3 is breaking unit tests for this plugin. 
Without downgrading we have about 16/48 (1 skipped) test suites passing.
With downgrading we have about 30/48 (1 skipped) test suites passing. 

I am working on other 17 tests which are breaking because of 2 components:
1. chrome.getUiSettingsClient()
2. chrome.breadcrumbs.set()

Reply:
@saratvemulapalli  so this PR doesn't fix all tests?

Reply:
No I am still working on fixing the others.

Reply:
All tests can pass now after fixing the chrome function mock issue ?

Reply:
All the tests are passing. 5 of them which are not, I created an issue to track them. #288